content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -53,10 +55,11 @@ You should define the following environment variables in your CI job:
     ...
     ```
 
-2.  `STUDIO_REPO_URL`: If your repository is on github.com, gitlab.com or
-    bitbucket.org, you do not need to set this environment variable. But if you
-    are using some other Git provider, then you should set the repository url in
-    this format: `{remote-type}:{namespace}/{repo-name}`. For example, for the
+2.  `STUDIO_REPO_URL`: If you are running the experiment locally, or your
```

Comment:
This one should be removed as the backend part of https://github.com/iterative/studio/issues/4801 is already merged and https://github.com/iterative/dvc-studio-client/pull/6 is getting merged soon.

Reply:
I'd say let's wait for those PRs to be merged. I can create a new docs PR with just this change.

Reply:
Let's not forget it. Should we create an issue?